apps/site/assets/ts/stop/components/amenities/AmenityCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import renderFa from "../../../helpers/render-fa";
+
+const AmenityCard = ({
+  headerText,
+  icon
+}: {
+  headerText: string;
+  icon: JSX.Element;
+}): JSX.Element => {
+  return (
+    <div className="d-flex card justify-content-space-between">
```

Comment:
I don't think using `. justify-content-space-between` makes sense here? Also it doesn't seem to change anything if I remove it.

Reply:
Justify content is used to make sure the chevron is on the right, and the content is on the left.  I believe you see no changes because of the placeholder text.  If that is removed the chevron will pull left next to the header of the card

Reply:
<img width="1234" alt="Screen Shot 2023-06-02 at 12 36 41 PM" src="https://github.com/mbta/dotcom/assets/2679229/6c5cf18f-8f5c-438f-8c32-83ef3c4213e8">

Reply:
Ah nice, I stand corrected!